src/meta/Matches.ts

```diff
@@ -0,0 +1,14 @@
+import { Base } from './Base';
+
+export default class Matches extends Base {
+	/**
+	 * Determine if the target of a particular `Event` matches the virtual DOM key
+	 * @param key The virtual DOM key
+	 * @param event The event object
+	 */
+	public get(key: string, event: Event): boolean {
+		this.requireNode(key);
```

Comment:
Ah, yes... I see, yeah, I guess it can't throw then, it needs to simply return `false`.  We can't get into a situation of trying to re-run a listener because a node wasn't found, just in case a re-render _might_ make it appear.  It needs to clearly just fail gracefully. 😖 It does highlight that we might need some sort of diagnostics for meta down the road to _expose_ what sort of nodes are being requested to identify unintentional failures.